renovate.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Renovate configuration for keras image similarity training repository",
+  "extends": [
+    "config:base",
+    ":dependencyDashboard",
+    ":rebaseStalePrs"
+  ],
+  "schedule": [
+    "before 3am every weekday"
+  ],
+  "enabledManagers": [
+    "github-actions",
+    "python",
+    "tensorflow"
```

Comment:
Can you point me to the docs for tensorflow and python as managers to make sure I'm not missing something? Seems like python might need to be replaced by one of the package managers like `pip_requirements`.
Not sure about tensorflow? 
Current supported managers are here I believe: 
https://docs.renovatebot.com/modules/manager/

Reply:
I referred to these below pointers about Tensorflow.
- https://repo1.dso.mil/dsop/opensource/tensorflow/tensorflow-2.2.0/-/blob/development/renovate.json#L13
- https://github.com/GoogleCloudPlatform/cloudml-samples/blob/main/renovate.json
For Python I took the reference from 
- https://github.com/GoogleCloudPlatform/cloudml-samples/blob/main/renovate.json

But I am fine to change it to `pip_requirements`

Reply:
I have added `tensorflow`  and `python` config in the `packageRules`. Please review again.